common/v2/components/WalletUnlock/Web3Provider.tsx

```diff
@@ -90,6 +88,48 @@ class Web3ProviderDecrypt extends Component<Props, State> {
       this.setState({ ...this.state, web3Unlocked: false });
     }
   }
+
+  private getWeb3Footer(strWeb3Name: string) {
+    switch (strWeb3Name.toUpperCase()) {
+      default:
+      case 'METAMASK':
+        return (
+          <div className="MetaMask-footer">
+            <div>
+              {translate(`ADD_ACCOUNT_${this.state.web3ProviderSettings.name}_FOOTER`)}{' '}
+              <NewTabLink
+                content={translate(
+                  `ADD_ACCOUNT_${this.state.web3ProviderSettings.name}_FOOTER_LINK`
+                )}
+                href="https://chrome.google.com/webstore/detail/metamask/nkbihfbeogaeaoehlefnkodbefgpgknn?hl=en"
+              />
+            </div>
+            <div>
+              <NewTabLink
+                content={translate('ADD_ACCOUNT_METAMASK_HELP')}
+                href={`${KB_URL}/how-to/migrating/moving-from-mycrypto-to-metamask`}
```

Comment:
I know it's copied from the previous, but it's a good time to start using `WALLETS_CONFIG` here.
```
import { WALLETS_CONFIG } from 'v2/config'
const WalletConfig = WALLETS_CONFIG[provider.name]
...
href={WalletConfig.helpLink}
```